server/src/main/java/dev/slimevr/Main.java

```diff
@@ -42,6 +44,10 @@ public static void main(String[] args) {
 			formatter.printHelp("slimevr.jar", options);
 			System.exit(0);
 		}
+		if (cmd.hasOption("version")) {
+			System.out.println("SlimeVR Server " + VERSION + "-" + BuildConfig.GIT_COMMIT_HASH);
+			System.exit(0);
+		}
```

Comment:
I know it's small, but could we just make a quick little method for printing the version instead of duplicating? 🥺

Reply:
Both prints are different, that goes directly to stdout and the other uses a logger :/

Reply:
Unless you are talking about a function for making that string, im gonna die XD

Reply:
Oh you're right, why isn't it using the logger? If it's intentionally not using the logger for some reason, then just resolve this